src/components/SectionCard/SectionCard.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import SectionCard from './SectionCard';
+import renderer from 'react-test-renderer';
+
+test('It calls the function that gets passed in as a prop to remove section on click.', () => {
+  const mockOnClick = jest.fn();
+
+  const props = {
+    title: 'This is a Section Title',
+    id: '98748345934',
+    lessons: [{}],
+  }
+
+  const sectionCard = shallow(<SectionCard {...props} removeSection={mockOnClick} />);
+  sectionCard.find('.deleteSection').simulate('click', { target: { value: 'Removes section' } }); 
```

Comment:
Quick Q. Does this Test actually test anything?
You are just finding a thing and simulating click. But you never check to see if `mockOnClick` was called.

Reply:
Nice catch - I didn't test to see if `mockOnClick` was called. Will fix this!